plugins/kubernetes/src/api/types.ts

```diff
@@ -49,3 +53,12 @@ export interface KubernetesApi {
     init?: RequestInit;
   }): Promise<Response>;
 }
+
+export interface KubernetesProxyApi {
+  getPodLogs(request: {
+    podName: string;
+    namespace: string;
+    clusterName: string;
+    containerName: string;
+  }): Promise<string>;
```

Comment:
```suggestion
  }): Promise<{ text: string }>;
```

Or something like that, makes it a lot easier to evolve this API in the future